src/Tizen.Multimedia/AudioManager/AudioDucking.cs

```diff
@@ -0,0 +1,146 @@
+        /// <param name="targetType">The type of sound stream to be affected by this new instance.</param>
+        /// <exception cref="ArgumentException"><paramref name="targetType"/> is invalid.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public AudioDucking(AudioStreamType targetType)
+        {
+            ValidationUtil.ValidateEnum(typeof(AudioStreamType), targetType, nameof(targetType));
+
+            _duckingStateChangedCallback = (AudioDuckingHandle ducking, bool isDucked, IntPtr _) =>
+            {
+                DuckingStateChanged?.Invoke(this,
+                    new AudioDuckingStateChangedEventArgs(IsDucked));
+            };
+
+            Interop.AudioDucking.Create(targetType, _duckingStateChangedCallback,
+                IntPtr.Zero, out _handle).ThrowIfError("Unable to create stream ducking");
+
+            Debug.Assert(_handle != null);
+        }
+
+        /// <summary>
+        /// Occurs when the ducking state is changed.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public event EventHandler<AudioDuckingStateChangedEventArgs> DuckingStateChanged;
+
+        /// <summary>
+        /// Gets the ducking state of the stream.
+        /// </summary>
+        /// <value>true if the audio stream is ducked; otherwise, false.</value>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsDucked
+        {
+            get
+            {
+                Interop.AudioDucking.IsDucked(Handle, out bool isDucked).
+                    ThrowIfError("Failed to get the running state of the device");
+
+                return isDucked;
+            }
+        }
+
+        /// <summary>
+        /// Activate audio ducking
+        /// </summary>
+        /// <param name="duration">The duration(milisecond) for ducking.</param>
+        /// <param name="ratio">The volume ratio after ducked.</param>
+        /// <exception cref="ArgumentOutOfRangeException">
+        ///     <paramref name="duration"/> is less than 0 or greater than 3000.<br/>
+        ///     -or-<br/>
+        ///     <paramref name="ratio"/> is less than 0.0 or greater than or equal to 1.0.<br/>
+        /// </exception>
+        /// <since_tizen> 6 </since_tizen>
+        public void Activate(uint duration, double ratio)
+        {
+            if (duration < 0 || duration > 3000)
+            {
+                throw new ArgumentOutOfRangeException(nameof(duration), duration, "Valid range : 0 <= duration <= 3000");
+            }
+
+            if (ratio < 0.0 || ratio >= 1.0)
+            {
+                throw new ArgumentOutOfRangeException(nameof(ratio), ratio, "Valid range : 0 <= ratio < 1.0");
+            }
+
+            Interop.AudioDucking.Activate(Handle, duration, ratio).
+                ThrowIfError("Failed to activate ducking");
+        }
+
+        /// <summary>
+        /// Deactivate audio ducking
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
```

Comment:
Same comments regarding exceptions of Activate() method.

Reply:
Done.